### ISSUE TITLE
Error adding interfaces to existing columns with camelCase naming

### ISSUE THREAD

**Issue by fl034** [directus/api#783]
# Bug Report

## Steps to Reproduce

1. Enable managing of existing MySQL (5.6) table.
2. Try to add interface (e.g. relation or datetime) to a field with a camelCase name
3. Getting error message `Column already exists: 1060 Duplicate column name "myColumnName"`

## Expected Behavior

Being able to use Directus on columns with camel case naming.

## Actual Behavior

UI shows nothing, but error in JS console.

## Other Context & Screenshots

![bildschirmfoto 2019-02-18 um 21 29 17](https://user-images.githubusercontent.com/11268968/52975211-e9c32e00-33c4-11e9-8850-61954a0f8d5b.png)

## Technical Details

- Web Server: [eg: Apache 2.4.37]
- PHP Version: 7.2
- Database: MySQL 5.6 
- Install Method: FTP-Upload of Directus 7.0.17

**Comment by benhaynes** [comment]
I assume the field you are trying to add is already in the database and you are just trying to "turn it on" within Directus?

It seems we would want to do something similar to tables/collections in the App... show any that already exist (disabled) and add a button for them to become "managed".

@rijkvanzanten thoughts?

**Comment by fl034** [comment]
> I assume the field you are trying to add is already in the database and you are just trying to "turn it on" within Directus?
Exactly. And it's working just fine for all other table columns that don't have camel case.

**Comment by benhaynes** [comment]
Interesting. Does the App let you type capital letters into the field name? I thought it "sanitized" them by making them all lowercase. If so, the fix would probably be a override to the "strict" naming filter for fields/collections.

In the meantime, you should be able to update this in the database directly within: `directus_fields`

**Comment by rijkvanzanten** [comment]
> Does the App let you type capital letters into the field name?

Curious to see if this is caused by the app, or if the API doesn't handle capital letters correctly

**Comment by fl034** [comment]
The app won't let me enter capital letters in the name field. But when I select an existing (camelcased) column, the name is shown lowercased and produces the error message above.

I'll try your workaround by editing the database directly :-)

**Comment by dev7ch** [comment]
I assume this could be solved quickly by remove `toLowerCase` in the app here:

https://github.com/directus/app/blob/master/src/components/modals/prompt.vue#L94

The DB name is saved in CamelCase (tested locally)

**Comment by dev7ch** [comment]
can this simply be removed ?
If you want i can send a PR.

**Comment by WellingGuzman** [comment]
This is related to the app, the API doesn't "clean" the field name, it create it as you send it.
